src/components/marquee-wizard/marquee-wizard.component.ts

```diff
@@ -28,6 +28,9 @@ export class MarqueeWizardComponent extends WizardComponent implements OnDestroy
     /** If set to true the resizable splitter will be enabled and set to the default width **/
     @Input() resizable: boolean = false;
 
+    /** Provide a custom template for the footer, positioned right of the previous/next buttons*/
```

Comment:
Could we re-use the `WizardFooterContext` interface and then remove this input as it should inherit this from the base class?